packages/turf-transform-rotate/index.js

```diff
@@ -0,0 +1,115 @@
+var helpers = require('@turf/helpers');
+var centroid = require('@turf/centroid');
+var invariant = require('@turf/invariant');
+var rhumbBearing = require('@turf/rhumb-bearing');
+var rhumbDistance = require('@turf/rhumb-distance');
+var rhumbDestination = require('@turf/rhumb-destination');
+var getCoord = invariant.getCoord;
+var getCoords = invariant.getCoords;
+var point = helpers.point;
+var feature = helpers.feature;
+var polygon = helpers.polygon;
+var multiPoint = helpers.multiPoint;
+var lineString = helpers.lineString;
+var multiPolygon = helpers.multiPolygon;
+var multiLineString = helpers.multiLineString;
+
+/**
+ * Rotates any geojson Feature or Geometry of a specified angle, around its `centroid` or a given `pivot` point;
+ * all rotations follow the right-hand rule: https://en.wikipedia.org/wiki/Right-hand_rule
+ *
+ * @name rotate
+ * @param {Geometry|Feature<any>} geojson object to be rotated
+ * @param {number} angle of rotation (along the vertical axis), from North in decimal degrees, negative clockwise
+ * @param {Geometry|Feature<Point>|Array<number>} [pivot=`centroid`] point around which the rotation will be performed
+ * @param {number} [xRotation=0] angle of rotation, respect to the horizontal (x,y) plain, about the x axis (not implemented);
```

Comment:
Giving it some more thought, I suppose rotating x and y could be the equivalent of rotating along just a latitude rhumb line or just a longitude rhumb line but the order of rotation would still be an issue